uniffi_bindgen/src/interface/mod.rs

```diff
@@ -754,7 +754,15 @@ impl ComponentInterface {
                     meta.index,
                 );
             }
-            cbi.methods.push(meta.into());
+            let defn: Method = meta.into();
+            for arg in &defn.arguments {
```

Comment:
I know this hasn't been done consistently everywhere else, but `defn.iter_types()` is a slightly more compact way of doing this.